### PR TITLE
Update twitch extension

### PR DESCRIPTION
**Pull request by Aayush9029**
## Description
  - Migrating to raycast version @1.26.3
  - Added better markdown support
  - Added detail view
  - Fixed streamlink errors
  - Fixed low latency streams
  - Removing uneccesary commands
  - Added Success and Error Toasts and HUDs
  - Added metadata for raycast store
  - Optimizing + Refactoring + Restucturing undelrying codebase
  - A lot of other small bug and QOL fixes + improvements

## Screencast
Metadata-1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
I was almost done doing the exact, awesome - could I help provide something?

I added the right avatar in my version :)

File: extensions/twitch/package.json [hunk 1]
```diff
@@ -1,143 +1,135 @@
...
+        {
+            "name": "following",
+            "title": "Search Following Channels",
+            "description": "Search following Channels",
+            "icon": "icon.png",
+            "mode": "view"
+        }
+    ],
+    "dependencies": {
+        "@raycast/api": "^1.26.3",
```
Comment (pernielsentikaer) [inline extensions/twitch/package.json:115]: You actually downgrading the API?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @Aayush9029 👋 

Looks like you didn't pull the latest version before doing this? A lot of work is rolled back 😞 

Could you please correct this 😄 💪

**Comment by Aayush9029** [comment]
Oh god! Didn't realize they updated it, I pushed local changes I made for myself😅

Will work on merging this

**Comment by pernielsentikaer** [comment]
It was actually me 😁 Let me send your some code a bit later 🤟

**Comment by Aayush9029** [comment]
@pernielsentikaer any updates 😄 
Looking forward to this a lot!

**Comment by Aayush9029** [comment]
@pernielsentikaer it's been a while 😭 still waiting for you

**Pull request closed without merging**